test/parallel_exec.jl

```diff
@@ -1093,3 +1093,36 @@ let
     ref = remotecall(bad_thunk, 2)
     @test_throws RemoteException fetch(ref)
 end
+
+# Test addprocs enable_threaded_blas parameter
+function check_remote_num_threads(processes_added, num_threads)
+    for proc_id in processes_added
+        remote_thread_count = remotecall_fetch(BLAS.get_num_threads, proc_id)
+        @test remote_thread_count == num_threads
+    end
+end
+
+try
+    BLAS.set_num_threads(4)
+    @test BLAS.get_num_threads() == 4
+    processes_added = addprocs(2)
+
+    @test BLAS.get_num_threads() == 4
+    # Threading disabled in children by default
+    check_remote_num_threads(processes_added, 1)
+
+    rmprocs(processes_added)
+
+    BLAS.set_num_threads(4)
+    @test BLAS.get_num_threads() == 4
+    processes_added = addprocs(2, enable_threaded_blas=true)
+    # Master thread default is 4
+    @test BLAS.get_num_threads() == 4
+    # Remote thread default is 2
+    check_remote_num_threads(processes_added, 2)
```

Comment:
I'm looking into a more robust way to do this.  It looks like the formula for picking the number of open BLAS threads is pretty complicated, and it's not guaranteed to be using open blas.  Right now I'm doing something that might be flakey: if the number of cores is > 2, assert the number of blas threads is >= 2.  Do you have any suggestions?